### ISSUE TITLE
KNN using tensorflow

### ISSUE THREAD

**Issue by Abhijit2505** [edualgo/Grokking-Machine-Learning#9]
Add a KNN model using Tensorflow framework.

## Steps to follow
Comment about your interest in the issue and the dataset you want to work with
Tell us your approach in brief
Let the mentor respond to you and only after that get started with the issue.
Don't forget to check the already existing model in order to know the structure.

Join the discussion channel here - [eduAlgo Community](https://edualgo-python-community.tribe.so/)
Let's have a chat at the community tab before starting contributions.

**Comment by sujal111** [comment]
Please allow me to work on this issue. 
Essentially , KNN approach will be used in such a fashion that the training data will be memorised and during the testing time, label of points will be returned
Iris dataset will be used here
